wallet/src/constants.js

```diff
@@ -0,0 +1,43 @@
+/* ignore unused exports */
+const DEFAULT_PRIVATE_KEY = '0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69e';
```

Comment:
As you pointed over the sprint call, this would be replaced by metamask wallet passing this key I am assuming

Reply:
Yes, this is a place holder for now.